lua/graph.lua

```diff
@@ -408,6 +408,17 @@ toll = {
 ["snowmobile"] = "true"
 }
 
+lit = {
+  ["yes"] = "true",
+  ["no"] = "false",
+  ["24/7"] = "true",
+  ["automatic"] = "true",
+  ["limited"] = "false",
```

Comment:
`lit=limited` is a special case that, according to the OSM wiki mainly occurs in Austria and Germany, where it signifies that a street light marked with a ["Laternenring"](https://de.wikipedia.org/wiki/Laternenring) is not turned on during the whole night, so I guess we cannot reliably set the marked edge as `lit=true`